definitions/tools/star_align_fusion.cwl

```diff
@@ -2,110 +2,125 @@
 class: CommandLineTool
 cwlVersion: v1.0
 label: "STAR: align reads to transcriptome"
-baseCommand: ["/usr/local/bin/STAR"]
+baseCommand: ["/bin/bash star.sh"]
 requirements:
     - class: ShellCommandRequirement
     - class: ResourceRequirement
       ramMin: 64000
       coresMin: 10
     - class: DockerRequirement
       dockerPull: "mgibio/star:2.7.0f"
-arguments: [
-    "--runThreadN", $(runtime.cores)
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'star.sh'
+        entry: |
+            
+            set -eou pipefail
+
+            passthrough_args=${@:1:25}
```

Comment:
It's probably safer to have all the "passthrough args" at the end.  That way you don't have to be sure to count them properly.  For example, `out_samtype` looks like it may produce a variable number of parameters depending how many array values are used.

Reply:
Yeah, a better solution here is to use named parameters - I was trying to do it with minimal edits.  I'll work on that.